File: dp_labs_and_solutions/dp_labs_and_solutions/dp_java_labs_and_solutions/bridge/lab/Square.java
// Square.java

class Square extends RightAngledQuadrilateral
{
	public Square(double aSide)
	{
		setSide(aSide);
	}

	public double getSide()
	{
		return mySide;
	}

	public void setSide(double aSide)
	{
		mySide = aSide;
	}

	private double mySide;
}

File: dp_labs_and_solutions/dp_labs_and_solutions/dp_cpp_labs_and_solutions/bridge/lab/square.cpp
// square.cpp

#include "square.h"

square::square(double a_side)
{
	set_side(a_side);
}

double square::get_side() const
{
	return my_side;
}

void square::set_side(double a_side)
{
	my_side = a_side;
}
